arangod/VocBase/LogicalCollection.cpp

```diff
@@ -403,13 +403,13 @@ ErrorCode LogicalCollection::getResponsibleShard(arangodb::velocypack::Slice sli
 /// @briefs creates a new document key, the input slice is ignored here
 std::string LogicalCollection::createKey(VPackSlice input) {
   if (isSatToSmartEdgeCollection() || isSmartToSatEdgeCollection()) {
-    return createSmartToSatKey(input);
+    return createKeyBetweenSmartAndSat(input);
   }
   return keyGenerator()->generate();
 }
 
 #ifndef USE_ENTERPRISE
-std::string LogicalCollection::createSmartToSatKey(VPackSlice) {
+std::string LogicalCollection::createKeyBetweenSmartAndSat(VPackSlice) const {
```

Comment:
```suggestion
std::string LogicalCollection::createKeyBetweenSmartAndSat(VPackSlice) {
```

Reply:
using keyGenerator is not allowed to be const. As @mchacki told me once, that the key generator uses / increments ticks to be able to create unique keys.